Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -150,3 +150,31 @@ QVector<QString> Settings::GetProfiles(const InputConfig* config) const
 
   return vec;
 }
+
+bool Settings::IsLogVisible() const
+{
+  return SConfig::GetInstance().m_InterfaceLogWindow;
```

Comment:
``Settings`` does not inherit ``QSettings`` anymore, right? So should I create a local QSettings instance or what?